src/jbuf/jbuf.c

```diff
@@ -89,8 +89,10 @@ static void frame_alloc(struct jbuf *jb, struct frame **f)
 		f0 = le->data;
 
 		STAT_INC(n_overflow);
+#if JBUF_STAT
```

Comment:
please remove this hunk

Reply:
jbuf::stat is only available with JBUF_STAT defined (via RELEASE=1).

Reply:
@alfredh Could you please have another look? Thanks!